plugins/pom.xml

```diff
@@ -45,6 +44,7 @@
         <module>rss</module>
         <module>mondrianinput</module>
         <module>avro-format</module>
+        <module>rest</module>
```

Comment:
This probably shouldn't be in lowdeps.  The reason we have a lowdeps and highdeps profile is because the plugins in highdeps have dependencies on either plugins in the lowdeps profile or other projects that get built before the highdeps profile that aren't in the kettle repository.  Since this now includes its metaverse analyzer, it's dependent on metaverse, which gets built before highdeps.  Having it in lowdeps will cause the build to fail.